tutorials/asr/05_Online_Noise_Augmentation.ipynb

```diff
@@ -706,6 +706,79 @@
     "ipd.Audio(sample_segment.samples, rate=16000)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## RIR and Noise Perturbation\n",
+    "RIR augmentation with additive foreground and background noise.\n",
+    "In this implementation audio data is augmented by first convolving the audio with a Room Impulse Response\n",
+    "and then adding foreground noise and background noise at various SNRs. RIR, foreground and background noises\n",
+    "should either be supplied with a manifest file or as tarred audio files (faster)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Prepare rir data and manifest"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rir_data_path = '/home/ubuntu/Data/asr'\n",
```

Comment:
Could you change this path to '.' 
You could reuse data_dir or change this to : 

```
# This is where the rir data will be downloaded.
# Change this if you don't want the data to be extracted in the current directory.
rir_data_path = '.' 
```
Thanks.

Reply:
It has been changed.